app/controllers/home_controller.rb

```diff
@@ -1,6 +1,8 @@
 # Controller to handle root HTTP
 # connections and display the home page
 class HomeController < ApplicationController
+  layout 'sul-requests'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.